lib/db.js

```diff
@@ -271,26 +276,21 @@ class Database extends EventEmitter {
    * @param {object} update - Document properties to update.
    * @param {object} [options] - Optional settings see mongo documentation
    * @param {boolean} [options.multiple=false] - Should multiple documents be updated.
-   * @param {boolean} [options.upsert=false] - Should documents be inserted if they don't already exist..
-   * @param {object} [options.writeConcern] - the write conern options https://mongodb.github.io/node-mongodb-native/4.0/classes/writeconcern.html
+   * @param {boolean} [options.upsert=false] - Should documents be inserted if they don't already exist.
+   * @param {object} [options.writeConcern] - the write concern options
    * @returns {Promise} resolves with an object with results, and ops as keys
    * @see {@link https://mongodb.github.io/node-mongodb-native/4.0/classes/collection.html#updateMany}
    * @see {@link https://mongodb.github.io/node-mongodb-native/4.0/classes/collection.html#updateOne}
+   * @see {@link https://mongodb.github.io/node-mongodb-native/4.0/interfaces/updateoptions.html#writeconcern}
    */
-  async update(collectionName, query, update, options) {
-    options = defaults(options, {
-      multiple: false,
-      upsert: false
-    });
-    debug('getting db collection');
+  async update(collectionName, query, update, options = {}) {
```

Comment:
default multiple and upsert to false?

Reply:
was just letting undefined be falsy - not a fan?